sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -60,6 +60,12 @@ class FnApiRunnerTest(unittest.TestCase):
   def create_pipeline(self):
     return beam.Pipeline(runner=fn_api_runner.FnApiRunner())
 
+  def create_pipeline_with_grpc(self):
```

Comment:
We should not be requiring GRPC for these tests. If it only works with the GRPC runner, please move these test to FnApiRunnerTestWithGrpc with a JIRA to fix this.